### DISCUSSION TITLE
Possible parallelization of get_descriptors() feature

### DISCUSSION THREAD

**Discussion by zarkoivkovicc** [ACEsuit/mace#385] [Q&A]
I would like to get descriptors for a list of molecules, and right now I have to call `calculator.get_descriptors(config)` for each molecule separately. It would be nice if we could put `list[configs]` and get a list of descriptors, in a way that each molecule is done in parallel (or in a relatively small amount of batches).

**Comment by ilyes319** [comment]
You can do that by adapting the eval config scripts to also write the descriptors.
